python_helper/setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import setuptools
+import os
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+with open(os.path.join(dir_path, 'README.md')) as f:
+    long_description = f.read()
+
+with open(os.path.join(dir_path, 'AUTHORS.txt')) as f:
+    authors = ', '.join([a.strip() for a in f.readlines()])
+
+setuptools.setup(
+    name='python_helper',
+    version='0.0.1',
+    description='Helper functions and classes for executing python unit tests',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author=authors,
+    author_email="sophia@cs.toronto.edu",
+    packages=['python_helper'],
+    install_requires=['timeout-decorator'],
```

Comment:
In addition to these, use `extras_require={'dev': ['hypothesis']}`, which allows developers for the package to install this package for running the test suite.

(e.g. see https://stackoverflow.com/questions/28509965/setuptools-development-requirements)